webapp/channels/src/components/new_search/search_box.tsx

```diff
@@ -228,7 +228,8 @@ const SearchBox = forwardRef(
         const changeSearchTeam = (selectedTeam: string) => {
             const newTerms = searchTerms.
                 replace(/\bin:[^\s]*/gi, '').replace(/\s{2,}/g, ' ').
```

Comment:
nit: it's outside the scope of this PR, but I'd add a comment explaining what the regexes do for clarity